app/src/main/java/com/babylon/wallet/android/presentation/transaction/analysis/TransactionAnalysisDelegate.kt

```diff
@@ -115,6 +115,7 @@ class TransactionAnalysisDelegate @Inject constructor(
         _state.update {
             it.copy(
                 isRawManifestVisible = previewType == PreviewType.NonConforming,
+                showRawTransactionWarning = previewType == PreviewType.NonConforming,
```

Comment:
maybe this property could be derived property of state? like 
`val showRawTransactionWarning
            get() = previewType == PreviewType.NonConforming`

Reply:
> maybe this property could be derived property of state? like
> `val showRawTransactionWarning get() = previewType == PreviewType.NonConforming`
> I don't think we need to declare new one

Yes you right 👍

Reply:
> > maybe this property could be derived property of state? like
> > `val showRawTransactionWarning get() = previewType == PreviewType.NonConforming`
> > I don't think we need to declare new one
> 
> Yes you right 👍

@jakub-rdx Edited: However I need a way to dismiss so I somehow need to reset its state, see `onAcknowledgeRawTransactionWarning` 
Is there a nice way you can see to do it or still I need some private val that will be used in the `showRawTransactionWarning` backing property ?